adapter-in/web/src/main/kotlin/com/yapp/bol/group/GroupController.kt

```diff
@@ -24,9 +24,9 @@ class GroupController(
 
     @GetMapping("/search")

```

Comment:
p4. 전체까지 된다면 그냥 Get 으로도 될 것 같아요
```suggestion
    @GetMapping

```

Reply:
> p4. 전체까지 된다면 그냥 Get 으로도 될 것 같아요

여기서 수정했습니다~
f5672fea642a94e5ea637e493b6d46ea2c10d678